Buddies/Controls/FilterSearchBar.swift

```diff
@@ -65,8 +67,24 @@ class FilterSearchBar : UISearchBar, UISearchBarDelegate {
         ])
     }
     
-    private func makeFilterButton(doing action: Selector, rounding corners: CACornerMask) -> UIButton {
-        let bttn = makeButton(saying: "Filter", doing: action)
+    func provideLoggedInUser(_ user: LoggedInUser?) {
```

Comment:
What is this terminology supposed to mean?  Is `provide` a common word for this context?  `setLoggedInUser` or similar seems clearer.

Reply:
idk, i think it's clear enough

Reply:
I would say provide is common. Also, it's not so much a setter as it is something that needs to be provided.